lib/ga-tag.js

```diff
@@ -2,6 +2,16 @@
 
 const _ = require('lodash');
 
+const convertToGTMPage = (text) => {
+  // Remove leading and trailing slashes
+  let str = text.replace(/^\/|\/$/g, '');
+  // Replace hyphens with spaces and capitalize each word
+  str = str.replace(/-+/g, ' ').replace(/(^|\s)\S/g, function(match) {
+    return match.toUpperCase();
+  });
+  return str;
+}
+
```

Comment:
Looks like github actions is complaining about some linting and styling

Reply:
aaah thanks ...god I hate linting 😂